src/site/_includes/js/test/eligibility-programs.test.js

```diff
@@ -592,6 +592,7 @@ describe('Program eligibility', () => {
       existingVaDisabilityHousehold: false,
       existingVtaParatransitMe: false,
       existingVtaParatransitHousehold: false,
+      existingRtcClipperMe: false,
```

Comment:
Even though it's not used, you should add `existingRtcClipperHousehold: false,` to be complete.

Reply:
Done in [f3038e7](https://github.com/theunitedeffort/theunitedeffort.org/pull/610/commits/f3038e71984f7e2cbbb53b7d59b5ede13d9315f8)